src/AsyncPoll.ts

```diff
@@ -42,6 +42,7 @@ export class AsyncPoll extends EventEmitter {
                 await callback();
             } catch (err) {
                 logger.error(`Error Executing AsyncPoll ${err}`);
+                logger.error(`Stack trace for error: ${err.stack}`);
```

Comment:
If we're going to leave this here, I'd prefer it be at the `debug` log level.

Reply:
Would you prefer to have it there or not? I thought it was useful, but might be better to move it to a more generic error handler

Reply:
Initially I was leaning towards not printing it at all for performance reasons as generating a stack trace in JS has typically been costly and avoided.  However, looking over the [V8 docs](https://v8.dev/docs/stack-trace-api), it appears the `err.stack` property is generated when the error is created, so including this trace in the log is not a performance hit.

Let's leave the call, but change the log level to `debug` and update the message to reference this being a trace for the AsyncPoll execution.